react-gbajs/src/emulator/gba.js

```diff
@@ -127,6 +122,11 @@ GameBoyAdvance.prototype.loadRomFromFile = function(romBuffer, callback) {
 }
 
 GameBoyAdvance.prototype.reset = function() {
+	if (this.rafId) {
+		window.cancelAnimationFrame(this.rafId);
+		delete this.rafId;
+	}
```

Comment:
even if not swtiching to RAF, canceling either RAF or setTimeout is needed here, otherwise after a restore, the gba runs at double speed.